kcidb/monitor/test_output.py

```diff
@@ -85,3 +85,89 @@ def test_min(self):
         self.assertEqual('utf-8', html.get_content_charset())
         content = html.get_payload()
         self.assertIn("We detected a new revision!", content)
+
+    def test_subject_and_body_length(self):
+        """Check subject and body length of Notification """
+
+        db_client = db.Client("sqlite::memory:")
+        db_client.init()
+        oo_client = oo.Client(db_client)
+        db_client.load({
+            "version": self.version,
+            "checkouts": [
+                {
+                    "contacts": [
+                        "rdma-dev-team@redhat.com"
+                    ],
+                    "start_time": "2020-03-02T15:16:15.790000+00:00",
+                    "git_repository_branch": "wip/jgg-for-next",
+                    "git_commit_hash": "5e29d1443c46b6ca70a4c940a67e8c09f05dcb7e",
+                    "patchset_hash": "",
+                    "git_repository_url": "https://git.kernel.org/pub/scm/linux/kernel/git/arm64/linux.git",
+                    "misc": {
+                        "pipeline_id": 467715
+                    },
+                    "id": "origin:1",
+                    "origin": "origin",
+                    "patchset_files": [],
+                    "valid": True,
+                },
+            ],
+            "builds": [
+                                {
+                    "id": "kernelci:kernelci.org:619fecfc6d932a49b5f2efb0",
+                    "checkout_id": "origin:1",
+                    "origin": "origin",
+                    "comment": "v5.16-rc2-19-g383a44aec91c",
+                    "start_time": "2021-11-25T20:07:24.499000+00:00",
+                    "duration": 486.695294857,
+                    "architecture": "x86_64",
+                    "config_name": "x86_64_defconfig+x86-chromebook+kselftest",
+                    "valid": True
+                },
+            ]
+        })
+        revision = oo_client.query(orm.Pattern.parse(">revision#"))["revision"][0]
+        message = NotificationMessage(
+            ["kernelci-results-staging@groups.io"],
+            subject="JINJA2_TEMPLATE_FOR_YOUR_SUBJECT",
+            body="JINJA2_TEMPLATE_FOR_YOUR_BODY"
```

Comment:
Please don't use the literal placeholders I put into the example :joy: I was only trying to illustrate the point for you, pointing out where subject/body go, and that these strings are in fact Jinja2 templates. Please put something less confusing there.

I find it useful to use strings of digits as samples when testing for length. Something like `"1234567890"` or `"123456789012345678901234"`. This way it's easier to see that the first string is ten characters long and the second is 24 characters. But pick whatever feels best for you.